rocket_twin/utils/run_sequences.py

```diff
@@ -0,0 +1,57 @@
+from cosapp.drivers import NonLinearSolver, RungeKutta
+from cosapp.recorders import DataFrameRecorder
+
+
+def run_sequences(sys, sequences):
+    """Run the command sequences over a system.
+
+    Inputs
+    ------
+    sys: System,
+        the system over which the commands are applied
+    sequences: dictionary,
+        the commands to be applied
+    """
+    rk = sys.add_driver(RungeKutta("rk"))
+    rk.add_recorder(DataFrameRecorder(hold=True))
+
+    for seq in sequences:
+        print("sequence ", seq["name"])
+
+        if seq["type"] == "transient":
+            sys.drivers.clear()
+            rk.children.clear()
+
+            sys.add_driver(rk)
+            run = rk.add_driver(NonLinearSolver("nls", tol=1e-6))
+            rk.time_interval = (rk.time, rk.time + 10000)
+
+            if "dt" in seq:
+                rk.dt = seq["dt"]
+
+            if "stop" in seq:
+                rk.set_scenario(stop=seq["stop"])
+
+        if seq["type"] == "static":
+            sys.drivers.clear()
+            run = sys.add_driver(NonLinearSolver("nls", tol=1e-6))
+
+        if "init" in seq:
+            for key, val in seq["init"].items():
+                sys[key] = val
+        if "Tf" in seq:
+            rk.time_interval = (rk.time, rk.time + seq["Tf"])
```

Comment:
this is a question of init, not a specific test here

Reply:
This one I don't really get it. If I want my sequence to run during a fixed time interval, where should I specify it?

Reply:
it looks more as a question of "stop" condition. what do you think about : 

stop = {'time_end' : 10000}

?

Reply:
Check